go-controller/pkg/ovn/policy_old_test.go

```diff
@@ -251,13 +251,6 @@ func (n networkPolicyOld) delPodCmds(fexec *ovntest.FakeExec, pod pod, networkPo
 	fexec.AddFakeCmdsNoOutputNoError([]string{
 		fmt.Sprintf("ovn-nbctl --timeout=15 clear address_set %s addresses", hashedNamespace),
 	})
-	fexec.AddFakeCmd(&ovntest.ExpectedCmd{
```

Comment:
Note that this is a drive-by fix to policy_old.go because it doesn't use port groups, but multicast depends on port groups, so we shouldn't be setting multicast enabled in policy_old.go at all.